Dockerfile

```diff
@@ -8,55 +8,67 @@ ENV CONF_VERSION  6.3.4
-        --delete               "Server/Service/Engine/Host/@debug" \
-        --delete               "Server/Service/Engine/Host/Context/@debug" \
-                               "${CONF_INSTALL}/conf/server.xml" \
-    && touch -d "@0"           "${CONF_INSTALL}/conf/server.xml" \
-    && chown daemon:daemon     "${JAVA_CACERTS}"
+    && mkdir -p                           "${CONF_HOME}" \
+    && chmod -R 700                       "${CONF_HOME}" \
+    && chown ${RUN_USER}:${RUN_GROUP}     "${CONF_HOME}" \
+    && mkdir -p                           "${CONF_INSTALL}/conf" \
+    && curl -Ls                           "${CONF_DOWNLOAD_URL}" | tar -xz --directory "${CONF_INSTALL}" --strip-components=1 --no-same-owner \
+    && curl -Ls                           "${MYSQL_DRIVER_DOWNLOAD_URL}" | tar -xz --directory "${CONF_INSTALL}/confluence/WEB-INF/lib" --strip-components=1 --no-same-owner "mysql-connector-java-${MYSQL_VERSION}/mysql-connector-java-${MYSQL_VERSION}-bin.jar" \
+    && chmod -R 700                       "${CONF_INSTALL}/conf" \
+    && chmod -R 700                       "${CONF_INSTALL}/temp" \
+    && chmod -R 700                       "${CONF_INSTALL}/logs" \
+    && chmod -R 700                       "${CONF_INSTALL}/work" \
+    && chown -R ${RUN_USER}:${RUN_GROUP}  "${CONF_INSTALL}/conf" \
+    && chown -R ${RUN_USER}:${RUN_GROUP}  "${CONF_INSTALL}/temp" \
+    && chown -R ${RUN_USER}:${RUN_GROUP}  "${CONF_INSTALL}/logs" \
+    && chown -R ${RUN_USER}:${RUN_GROUP}  "${CONF_INSTALL}/work" \
+    && echo -e                            "\nconfluence.home=${CONF_HOME}" >> "${CONF_INSTALL}/confluence/WEB-INF/classes/confluence-init.properties" \
+    && xmlstarlet                         ed --inplace \
+        --delete                          "Server/@debug" \
+        --delete                          "Server/Service/Connector/@debug" \
+        --delete                          "Server/Service/Connector/@useURIValidationHack" \
+        --delete                          "Server/Service/Connector/@minProcessors" \
+        --delete                          "Server/Service/Connector/@maxProcessors" \
+        --delete                          "Server/Service/Engine/@debug" \
+        --delete                          "Server/Service/Engine/Host/@debug" \
+        --delete                          "Server/Service/Engine/Host/Context/@debug" \
+                                          "${CONF_INSTALL}/conf/server.xml" \
+    && touch -d "@0"                      "${CONF_INSTALL}/conf/server.xml" \
+    && chown ${RUN_USER}:${RUN_GROUP}     "${JAVA_CACERTS}"
 
 # Use the default unprivileged account. This could be considered bad practice
 # on systems where multiple processes end up being executed by 'daemon' but
 # here we only ever run one process anyway.
-USER daemon:daemon
+USER ${RUN_USER}:${RUN_GROUP}
 
 # Expose default HTTP connector port.
 EXPOSE 8090
+EXPOSE 8091
```

Comment:
Should README file be updated with this information ?

Reply:
Yep @netflash . Fixed.